beacon-chain/rpc/prysm/v1alpha1/beacon/van_pending_blocks_test.go

```diff
@@ -96,7 +97,9 @@ func TestServer_StreamNewPendingBlocks_PublishBlocks(t *testing.T) {
 		StateNotifier: chainService.StateNotifier(),
 		BlockNotifier: chainService.BlockNotifier(),
 		FinalizationFetcher: &chainMock.ChainService{
-			FinalizedCheckPoint: s.FinalizedCheckpoint()},
+			FinalizedCheckPoint: s.FinalizedCheckpoint(),
+		},
+		SyncChecker: &mockSync.Sync{IsSyncing: true},
```

Comment:
we miss a test case for `IsSyncing: false` and check the sideeffect on both cases